scripts/secnetperf-helpers.psm1

```diff
@@ -222,6 +222,14 @@ function Cleanup-State {
             if ($null -ne (Get-Service msquicpriv -ErrorAction Ignore)) { throw "msquicpriv still running remotely!" }
         }
     }
+
+    # Clean up any ETL residue.
```

Comment:
Another thought: Should we do this **before** running tests in addition? In case a previous test crashed...

Reply:
I don't think we need to go that far. This only became a problem after accumulating ETLs for months, and with this PR we would always cleanup every run. Making it every test I don't think is necessary.

Reply:
We only clean up if previous doesn't crash. If you run a test and it crashes, you never cleanup.

Reply:
That's true, I meant I feel like its ok to let a little residual buildup. If you run a test, it crashes, you don't cleanup, but the buildup is minimal (this only became a problem after accumulating ETLs for a long time). Then on the next workflow run, it cleans up whatever is left from the last. I don't want this to be a blocker, though. I can add that additional cleanup as well.

Reply:
We can start with this and go from there.